client/my-sites/domains/domain-management/edit-contact-info-page/index.tsx

```diff
@@ -1,19 +1,176 @@
+	const translate = useTranslate();
+
+	const isDataLoading = () => {
+		return ! getSelectedDomain( { domains, selectedDomainName } ) || isRequestingWhois;
+	};
+
+	const goToContactsPrivacy = () => {
+		page( domainManagementContactsPrivacy( selectedSite?.slug, selectedDomainName, currentRoute ) );
+	};
+
+	const renderBreadcrumbs = () => {
+		const previousPath = domainManagementEdit(
+			selectedSite?.slug,
+			selectedDomainName,
+			currentRoute
+		);
+
+		const items = [
+			{
+				label: translate( 'Domains' ),
+				href: domainManagementList( selectedSite?.slug, currentRoute ),
+			},
+			{
+				label: selectedDomainName,
+				href: previousPath,
+			},
+			{ label: translate( 'Edit contact infomation' ) },
+		];
+
+		const mobileItem = {
+			label: translate( 'Back' ),
+			href: previousPath,
+			showBackArrow: true,
+		};
+
+		return <Breadcrumbs items={ items } mobileItem={ mobileItem } />;
+	};
+
+	const renderContent = () => {
+		const domain = getSelectedDomain( { domains, selectedDomainName } );
+
+		if ( ! domain.currentUserCanManage ) {
+			return <NonOwnerCard domains={ domains } selectedDomainName={ selectedDomainName } />;
+		}
+
+		if ( domain.isPendingWhoisUpdate ) {
+			return <PendingWhoisUpdateCard />;
+		}
+
+		if ( domain.mustRemovePrivacyBeforeContactUpdate && domain.privateDomain ) {
+			return (
+				<EditContactInfoPrivacyEnabledCard
+					selectedDomainName={ selectedDomainName }
+					selectedSiteSlug={ selectedSite?.slug as string }
+				/>
+			);
+		}
+
+		return (
+			<EditContactInfoFormCard
+				domainRegistrationAgreementUrl={ domain.domainRegistrationAgreementUrl }
+				selectedDomain={ domain }
+				selectedSite={ selectedSite }
+				showContactInfoNote={ false }
+			/>
+		);
+	};
+
+	const renderSidebar = () => {
+		const supportLink = (
+			<ExternalLink
+				href={ localizeUrl(
+					'https://wordpress.com/support/domains/domain-registrations-and-privacy/#privacy-protection'
+				) }
+				target="_blank"
+				icon={ false }
+			/>
+		);
+		const icannLink = (
+			<ExternalLink
+				href="https://www.icann.org/resources/pages/contact-verification-2013-05-03-en"
+				target="_blank"
+				icon={ false }
+			/>
+		);
+		const explanationText1 = translate(
+			'{{icannLinkComponent}}ICANN{{/icannLinkComponent}} requires accurate contact information for registrants. This information will be validated after purchase. Failure to validate your contact information will result in domain suspension.',
+			{
+				components: {
+					icannLinkComponent: icannLink,
+				},
+			}
+		);
+		const explanationText2 = translate(
+			'Domain privacy service is included for free on applicable domains. {{supportLinkComponent}}Learn more{{/supportLinkComponent}}.',
+			{
+				components: {
+					supportLinkComponent: supportLink,
+				},
+			}
+		);
+		return (
+			<div className="edit-contact-info-page__sidebar">
+				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace */ }
```

Comment:
Is there a specific reason for disabling the `wpcalypso/jsx-classname-namespace` rule? 
The same result could be achieved if you use a classname ending with `-title` instead of `__title` - and it would be aligned with BEM conventions, avoiding the rule bypass 😸

Reply:
Cool! I'm not really into BEM, I thought we should use the same syntax even for subcomponents 😅. Good to know, thanks!